runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4001,23 +4001,51 @@ class INTERPRETER_CLASS
 		return EXECUTE_BYTECODE;
 	}
 
-	/* ToDo: unimplemented - https://github.com/eclipse-openj9/openj9/issues/13614 */
 	/* jdk.internal.misc.Unsafe: public native <V> V uninitializedDefaultValue(Class<?> clz); */
 	VMINLINE VM_BytecodeAction
 	inlUnsafeUninitializedDefaultValue(REGISTER_ARGS_LIST)
 	{
+		j9object_t cls = *(j9object_t*)_sp;
+
 		updateVMStruct(REGISTER_ARGS);
-		Assert_VM_unreachable();
+		buildInternalNativeStackFrame(REGISTER_ARGS);
+
+		j9object_t result = NULL;
+
+		/* ToDo: update this function to have the same behavior as OpenJDK when cls is null or not a vlauetype (currently OpenJDK segfaults in both those scenarios) */
```

Comment:
use `TODO` insteaad of `ToDo`. Also, for TODOs please open an issue so it can be tracked

Reply:
Okay, since there seems to be a few places where OpenJDK is segfaulting on unsafe APIs should I open a single issue for all of them?

Reply:
yes a single issue is fine. You can put the issue number in the comments

Reply:
okay, I made #14073

Reply:
the comments have been addressed in 0ee0b36